[Doc] Document the usage of LS_JAVA_OPTS environment variable (#14738)
elastic/logstash: docs/static/config-details.asciidoc

[[jvm-settings]]
=== JVM settings

> Configure the jvm settings in the `jvm.options` <<settings-files,settings file>> or with the <<ls-java-opts, `LS_JAVA_OPTS`>>.

Reviewer: I'm not sure 

```suggestion
Configure JVM settings in the `jvm.options` <<settings-files,settings file>>. JVM settings can also be set via the <<ls-java-opts, `LS_JAVA_OPTS`>> environment variable
```

Author: Far better, my `or` wasn't an exclusive or, but this way sounds more explicit!

> Configure JVM settings in the `jvm.options` <<settings-files,settings file>>. JVM settings can also be set via the <<ls-java-opts, `LS_JAVA_OPTS`>> environment variable

Reviewer: I missed a full stop and a line break from my original suggestion 🤦 

```suggestion
Configure JVM settings in the `jvm.options` <<settings-files,settings file>>. JVM settings can also be set via the <<ls-java-opts, `LS_JAVA_OPTS`>> environment variable.

```

Depending on the default stack size, start by multiplying by 4x, then 8x, and
then 16x until the overflow error resolves.

[[ls-java-opts]]
==== Using `LS_JAVA_OPTS`

> `LS_JAVA_OPTS` environment variable could be used to configure jvm options without editing the `jvm.options` <<settings-files,settings file>>.

Reviewer: ```suggestion
The `LS_JAVA_OPTS` environment variable can also be used to override JVM settings in the `jvm.options` file <<settings-files,settings file>>.
```

> The content of the variable is combined with the one present in the `jvm.options`.

Reviewer: ```suggestion
The content of this variable is additive to options configured in the `jvm.options` file, and will override any settings that exist in both places
```

> The content of this variable is additive to options configured in the `jvm.options` file, and will override any settings that exist in both places

Reviewer: I missed a full stop from my original suggestion 🤦 

```suggestion
The content of this variable is additive to options configured in the `jvm.options` file, and will override any settings that exist in both places.
```
